MediaBrowser.Common/Net/NetworkExtensions.cs

```diff
@@ -45,7 +45,7 @@ public static void AddItem(this NetCollection source, IPObject item)
         /// </summary>
         /// <param name="source">The <see cref="NetCollection"/>.</param>
         /// <returns>Returns a string representation of this object.</returns>
-        public static string Readable(this NetCollection source)
+        public static string AsString(this NetCollection source)
```

Comment:
DISCLAIMER: this is the first thing that came to my mind, but i have not checked the code if this is possible.

Why not call this in a overwritten toString or have the whole logic of the method in a overwritten toString?

Reply:
Originally all this code was in the Netcollection class which was an inheritence of Collection. However, creating an inheritence was frowned upon - hence - in an attempt to get it accepted, i moved the code to the network extension class.